schemaregistry/schemaregistry.md

```diff
@@ -405,3 +497,196 @@ The returned ´Content-Type´ is the same that was passed when the schema versio
 was registered.
 
 The HEAD method SHOULD also be implemented.
+
+#### 3.4.3. Get a specific schema version by schema version URI
+
+Each schema version has a URI, as defined in [2.2.2](#222-schema-version-uri).
+
+This operation is a GET on `/schema?uri={uri}`, with the required parameter
+being the schema version URI.
+
+The returned payload is the schema document. Further attributes such as the
+`description` and the `format` indicator are returned as HTTP headers.
+
+The returned ´Content-Type´ is the same that was passed when the schema version
+was registered.
+
+The HEAD method SHOULD also be implemented.
+
+## 4. Replication Model and State Change Events
+
+Many eventing scenarios require that events cross from private networks into
+public networks (and possibly back into a different private network) and it is
+quite common that event producers and event consumers do not share the same
+network scope.
+
+In complex systems made up of subsystems owned by different organizations, the
+subsystem owners might also want to decouple the systems such that one or not
+dependent on the other system's availability whereever possible, and therefore
+introduce redundant footprint for metadata stores such as a schema registry.
+
+We therefore cannot assume that an event consumer has access to the same
+endpoints accessible to the producer. The producer might use a schema registry
+endpoint to publish schemas that is different from the endpoint from which the
+consumer will need to fetch those schemas, and schemas will be synchronized
+between these distinct registries, parallel to the event flow.
+
+For the discussion in this section, we will use the term "source" for a registry
+from which schemas originate and "target" for a registry into which those
+schemas shall be added.
+
+The synchronization is accomplished by a combination of two mechanisms:
+
+1. The Schema Registry API explained in [section 3](#3-http-rest-api) and
+   formally defined in the [OpenAPI spec](schemaregistry.yaml) allows for
+   schema information to be read from a source and written imperatively to a
+   target: A target can pull changes or a source can push changes.
+2. A set of change events, defined in this section, can be emitted by the source
+   whenever any aspect of its state changes, and the target can update the
+   replica of said state by subscribing to these events.
+
+### 4.1 Producer authority or central authority
```

Comment:
If we wanted to support replication for Discovery Endpoints, do you think we'd end up using a similar mechanism to what you've defined in this PR? If so, could these concepts be abstracted into a common spec/doc?

Reply:
As discussed, I think we should copy the mechanism, but a standalone abstract spec seems odd.